examples/gno.land/r/sys/validators/v2/poc.gno

```diff
@@ -16,18 +41,54 @@ func NewPropRequest(changesFn func() []validators.Validator, title, description
 	}
 
 	callback := func(cur realm) error {
+		// Enforce cooldown: proposals can only update the validator set after the cooldown period.
+		if time.Since(lastValsetUpdate) < valsetUpdateCooldown {
```

Comment:
I'm trying to figure out if this check should be enforced outside the callback, when the user is creating the proposal

The concern being we can create a prop -> we vote -> we try to execute -> the proposal fails because of the cooldown

we then can't re-execute the prop because it failed, so we need to run a new one

Also keep in mind anyone from the GovDAO can execute the proposal once it's accepted